src/applications/proxy-rewrite/redirects/crossDomainRedirects.json

```diff
@@ -433,5 +433,50 @@
     "domain": "www.benefits.va.gov",
     "src": "/compensation/resources-rates-read-dic.asp",
     "dest": "/disability/survivor-dic-rates/"
+  },
+  {
+    "domain": "www.benefits.va.gov",
+    "src": "/homeloans/veteran_borrowers.asp",
+    "dest": "/housing-assistance/home-loans/"
+  },
+  {
+    "domain": "www.benefits.va.gov",
+    "src": "/homeloans/purchasecashout.asp",
+    "dest": "/housing-assistance/home-loans/loan-types/"
+  },
+  {
+    "domain": "www.benefits.va.gov",
+    "src": "/homeloans/purchaseco_buy_process.asp",
+    "dest": "/housing-assistance/home-loans/home-buying-process/"
+  },
+  {
+    "domain": "www.benefits.va.gov",
+    "src": "/homeloans/nadl.asp",
+    "dest": "/housing-assistance/home-loans/loan-types/native-american-direct-loan/"
+  },
+  {
+    "domain": "www.benefits.va.gov",
+    "src": "/homeloans/irrrl.asp",
```

Comment:
3 `r`s? Just want to confirm this isn't a typo 🙂

Reply:
Ah nope! Looks like that is an actual page.

Reply:
You don't think `irrrl` is descriptive? 😆